marlowe-dashboard-client/src/Dashboard/View.purs

```diff
@@ -49,6 +49,8 @@ dashboardScreen { currentSlot, tzOffset } state =
 
     walletDetails = state ^. _walletDetails
 
+    lastMarloweAppEndpointCall = state ^. _lastMarloweAppEndpointCall
```

Comment:
More curious than anything - why do we use the lenses to access the record properties like this?  What is the benefit of using lenses like this, rather than just

```purs
lastMarloweAppEndpointCall = state.lastMarloweAppEndpointCall
```

Or even 

```purs
{ lastMarloweAppEndpointCall } = state
```

Reply:
For simple record access, almost none, and they do introduce some performance overhead. 

I'd say that the benefit would be consistency. And in other more complex places it allows you to easily refactor if you want to change a record for a newtype, you only change the lens and the usage remains the same.

I'm culty of adding some record syntax from time to time 😅

Reply:
I've tried to make a point of using them everywhere, mainly for the second reason Hernán gave (if we ever need to change to a newtype, we just change the lens and we're done), and also partly so I don't waste time trying to decide if I should be using a lens or not each time... But I probably haven't been entirely consistent. :shrug:

Reply:
That is a good point. Maybe what we should be aiming for consistency on is wrapping all record types in newtype wrappers. That way we encapsulate the implementation details of the type, preventing their internals getting exposed all over the application... this makes refactoring much more difficult.

Reply:
I'm broadly in favour of using newtypes everywhere. It's a little extra boilerplate, but then you get repaid for you efforts with nicer error messages. But that'll be up to you folks now... 🤷